tests/ops/qubit/test_state_prep.py

```diff
@@ -62,6 +62,7 @@ class TestOperations:
         [
             qml.BasisState(np.array([0, 1]), wires=0),
             qml.QubitStateVector(np.array([1.0, 0.0]), wires=0),
+            qml.QubitDensityMatrix(np.array([[1.0, 0], [0.0, 0.0]]), wires=0),
```

Comment:
Just for consistency 👍🏼 
```suggestion
            qml.QubitDensityMatrix(np.array([[1.0, 0.0], [0.0, 0.0]]), wires=0),
```